### PR TITLE
Add recipe for r-multiblock

### PR DESCRIPTION
**Pull request by RJMW**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by RJMW** [comment]
https://github.com/orgs/conda-forge/teams/help-r, ready for review!

Missing dependencies added:
https://github.com/conda-forge/staged-recipes/pull/23536
https://github.com/conda-forge/staged-recipes/pull/23537

**Comment by RJMW** [comment]
@conda-forge/help-r, ready for review!

Missing dependencies added:
https://github.com/conda-forge/staged-recipes/pull/23536
https://github.com/conda-forge/staged-recipes/pull/23537

**Comment by RJMW** [comment]
@conda-forge-admin, please restart ci

**Comment by bgruening** [comment]
>   libGL.so.1: cannot open shared object file: No such file or directory

Looks like you are missing some dependencies/CDTs.

**Comment by RJMW** [comment]
Hi @bgruening @dbast

Rbase v4.3.x is not available for Windows. I have add a skip for the windows build.

File: recipes/r-multiblock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    - r-progress
+
+test:
+  commands:
+    - $R -e "library('multiblock')"           # [not win]
+    - "\"%R%\" -e \"library('multiblock')\""  # [win]
+
+about:
+  home: https://khliland.github.io/multiblock/, https://github.com/khliland/multiblock/
+  license: GPL-2.0-only
```
Comment (dbast) [inline recipes/r-multiblock/meta.yaml:83]: `# License: GPL (>= 2)` maps to
```suggestion
  license: GPL-2.0-or-later
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**